roundabout-core/src/test/java/at/fhv/itm3/s2/roundabout/util/ConfigParserTest.java

```diff
@@ -23,7 +23,7 @@ public void setUp() throws ConfigParserException {
     @Test
     public void configParserTest_parametersLoaded() {
         assertNotNull("parameters attribute not null", roundAboutConfig.getParameters());
-        assertEquals("6 parameters loaded", 6, roundAboutConfig.getParameters().getParameter().size());
+        assertEquals("6 parameters loaded", 1, roundAboutConfig.getParameters().getParameter().size());
```

Comment:
here you see the bad result of adding a message: you changed the code but not the message